README.md

```diff
@@ -237,388 +104,96 @@ You need to copy the following files:
-(found in the Git repository under /src/lang) and translate.
-
-You must change the first two lines of the file appropriately:
-
-    ##name English-Name-Of-Language
-    ##ownname Native-Name-Of-Language
-
-Note: Do not alter the following parts of the file:
-
-- String identifiers (the first word on each line)
-- Parts of the strings which are in curly braces (such as {STRING})
-- Lines beginning with ## (such as ##id), other than the first two lines
-   of the file
-
-### 7.2) Previewing
-
-In order to view the translation in the game, you need to compile your language
-file with the strgen utility. As this utility is tailored to a specific OpenTTD
-version, you need to compile it yourself. Just take the normal OpenTTD sources
-and build that. During the build process the strgen utility will be made.
-
-strgen is a command-line utility. It takes the language filename as parameter.
-
-Example:
-
-    strgen lang/german.txt
-
-This results in compiling german.txt and produces another file named german.lng.
-Any missing strings are replaced with the English strings. Note that it looks
-for english.txt in the lang subdirectory, which is where your language file
-should also be.
-
-That is all! You should now be able to select the language in the game options.
-
-## 8.0) Troubleshooting
-
-To see all startup options available to you, start OpenTTD with the
-'`./openttd -h`' option. This might help you tweak some of the settings.
-
-If the game is acting strange and you feel adventurous you can try the
-'`-d [[<name>=]<level>]`' flag, where the higher levels will give you more
-debugging output. The 'name' variable can help you to display only some type of
-debugging messages. This is mostly undocumented so best is to look in the
-source code file debug.c for the various debugging types. For more information
-look at https://wiki.openttd.org/Command_line.
 
-The most frequent problem is missing data files. Please install OpenGFX and
-possibly OpenSFX and OpenMSX. See section 4.1.1 for more information.
+## 1.5) Add-on content / mods
+
+### 1.5.1) AI opponents
+
+OpenTTD comes without AI opponents, so if you want to play with AIs you have to download them.
+
+The easiest way is via the 'Check Online Content' button in the main menu.
+
+You can select some AIs that you think are compatible with your playing style.
+
+AI help and discussions may also be found in the [AI section of TT-Forums](https://www.tt-forums.net/viewforum.php?f=65).
+
+### 1.5.2) Scenarios and height maps
+
+Scenarios and heightmaps can be added via the 'Check Online Content' button in the main menu.
+
+### 1.5.3) NewGRFs
 
-Under certain circumstance, especially on Ubuntu OpenTTD can be extremely slow
-and/or freeze. See known-bugs.txt for more information and how to solve this
-problem on your computer.
+A wide range of add-content is available as NewGRFs, including vehicles, industries, stations, landscape objects, town names and more.
```

Comment:
I think this can be elaborated a bit more

Reply:
A case could also be made for a 1 paragraph overview of all add-on content in OpenTTD, under the 'Add-on content / mods' heading, but I couldn't think what to write there.

Reply:
"is usually straightforward to install." is tautological nonsense that has no place in a readme. people who have no trouble installing won't ever read this sentence (because why would you read a readme if you don't have a problem), and people who have problems installing will be mocked by this sentence.